src/devices/src/virtio/net/unixgram.rs

```diff
@@ -80,11 +87,11 @@ impl Gvproxy {
             getsockopt(fd, sockopt::RcvBuf)
         );
 
-        Ok(Self { fd })
+        Ok(Self::new(fd))
     }
 }
 
-impl NetBackend for Gvproxy {
+impl NetBackend for Unixgram {
     /// Try to read a frame from passt. If no bytes are available reports ReadError::NothingRead
```

Comment:
We can replace the leftover "passt" with "peer"

Reply:
I've replaced them with "network proxy" or "proxy".